vaas-app/src/vaas/cluster/cluster.py

```diff
@@ -154,6 +154,10 @@ def _append_vcl(self, vcl, server, future_results, executor):
         loader = VclLoader(self.api_provider.get_api(server), server.status == 'maintenance')
         future_results.append(tuple([vcl, loader, server, executor.submit(loader.load_new_vcl, vcl)]))
 
+    def _discard_vcl(self, server, future_results, executor):
+        loader = VclLoader(self.api_provider.get_api(server), server.status == 'maintenance')
```

Comment:
Try to eliminate code duplication (creating instance of VclLoader - could be done once)